### PR TITLE
tickets/PREOPS-1262

### PR DESCRIPTION
**Pull request by MelissaGraham**
Update NB 02, our intermediate TAP queries notebook.
It used to rely heavily on the truth table, but that has all been updated now.
Requesting a review from @leannep , the original NB author.

### CODE REVIEW

**Comment by MelissaGraham** [comment]
Note: reviewer can approve at data-int.lsst.cloud, but this will not be merged until it has been confirmed to run at data.lsst.cloud.

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 1]
```diff
@@ -0,0 +1,1358 @@
...
+    "plt.title('objectId = ' + str(use_objectId))\n",
+    "plt.legend(loc='lower left')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Whether or not that random Object is truly variable is left as an exercise for the learner.\n",
```
Comment (leannep) [inline 02_Catalog_Queries_with_TAP.ipynb:643]: randomly choosen Object

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 2]
```diff
@@ -0,0 +1,1358 @@
...
+    "        \"ON fs.objectId = obj.objectId \" + \\\n",
+    "        \"WHERE fs.objectId = \" + str(use_objectId) + \" \" + \\\n",
+    "        \"AND fs.band = 'r' \""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 4. Visualize results that generate large data sets\n",
```
Comment (leannep) [inline 02_Catalog_Queries_with_TAP.ipynb:679]: results don't generate datasets - querys do -> "Visualize  large data sets resulting from a query"

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 3]
```diff
@@ -0,0 +1,1358 @@
...
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "temp = use_center_coords.split(', ')\n",
+    "center_ra = float(temp[0])\n",
+    "center_dec = float(temp[1])\n",
+    "print(center_ra, center_dec)"
+   ]
```
Comment (leannep) [inline 02_Catalog_Queries_with_TAP.ipynb:773]: This is not necessary if you use an astropy SkyCoord object - you can avoid sting splits and temp objects.

**Review by leannep — CHANGES_REQUESTED** [review]

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 4]
```diff
@@ -0,0 +1,1358 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pandas.set_option('display.max_rows', 20)\n",
+    "\n",
+    "output_notebook()\n",
+    "\n",
+    "warnings.simplefilter(\"ignore\", category=UnitsWarning)"
+   ]
+  },
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply 02_Catalog_Queries_with_TAP.ipynb:131]: no reason, will remove

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 5]
```diff
@@ -0,0 +1,1358 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "use_center_coords = \"62, -37\"\n",
+    "use_radius = \"1.0\""
+   ]
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply 02_Catalog_Queries_with_TAP.ipynb:377]: added back

File: 02_Catalog_Queries_with_TAP.ipynb [hunk 6]
```diff
@@ -0,0 +1,1358 @@
...
+   "outputs": [],
+   "source": [
+    "search_string = 'coord'\n",
+    "\n",
+    "for cname in results['column_name']:\n",
+    "    temp = str(cname)\n",
+    "    if temp.find(search_string) > -1:\n",
+    "        print(temp)\n",
+    "    del temp"
+   ]
```
[the remark this answers is not in the document]
Comment (MelissaGraham) [reply 02_Catalog_Queries_with_TAP.ipynb:330]: updated

File: 02_Catalog_Queries_with_TAP.ipynb (hunk 2 above)
Comment (MelissaGraham) [reply 02_Catalog_Queries_with_TAP.ipynb:679]: fixed

File: 02_Catalog_Queries_with_TAP.ipynb (hunk 3 above)
Comment (MelissaGraham) [reply 02_Catalog_Queries_with_TAP.ipynb:773]: fixed

**Pull request merged**